components.yaml

```diff
@@ -24,11 +24,10 @@ NCEP_Shared:
 GMAO_Shared:
   local: ./ESMF/Shared/GMAO_Shared@
   remote: ../GMAO_Shared.git
-  tag: v1.1.4
   sparse: ./config/GMAO_Shared.sparse
-  develop: master
+  branch: feature/wjamieson/GOCARTrefactor
```

Comment:
We need to tag this and reference a tag.   Probably should issue a release.  Let's discuss with @mathomp4.

Reply:
main on GMAO_Shared may now work just fine.    @WilliamJamieson will test.